tests/playwright/assets/api-requests.ts

```diff
@@ -81,13 +81,13 @@ export default class ApiRequests {
 		await Promise.all( requests );
 	}
 
-	public async cleanUpTestPages( request: APIRequestContext ) {
+	public async cleanUpTestPages( request: APIRequestContext, shouldDeleteAllPages = false ) {
 		const pagesPublished = await this.getPages( request ),
 			pagesDraft = await this.getPages( request, 'draft' ),
 			pages = [ ...pagesPublished, ...pagesDraft ];
 
 		const pageIds = pages
-			.filter( ( page: WpPage ) => page.title.rendered.includes( 'Playwright Test Page' ) )
+			.filter( ( page: WpPage ) => shouldDeleteAllPages || page.title.rendered.includes( 'Playwright Test Page' ) )
```

Comment:
I added this to allow deletion of all pages to have a clean slate for the checklist's create pages step